lgc/patch/PatchLoopMetadata.cpp

```diff
@@ -70,6 +76,38 @@ class PatchLoopMetadata : public Patch {
 // Initializes static members.
 char PatchLoopMetadata::ID = 0;
 
+// =====================================================================================================================
+// Update metadata by removing any existing metadata with the specified prefix, and then adding the new metadata if
+// existing metadata was removed or conditional is false.
+MDNode *PatchLoopMetadata::updateMetadata(MDNode *LoopID, ArrayRef<StringRef> RemovePrefixes, Metadata *newMetadata,
```

Comment:
nit: maybe RemovePrefixes -> PrefixesToRemove or DiscardedPrefixes/DroppedPrefixes? The imperative style of RemovePrefixes makes it look more like a function name to me

Reply:
Changed to prefixesToRemove.